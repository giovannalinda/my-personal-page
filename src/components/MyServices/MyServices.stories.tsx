import { Meta, Story } from '@storybook/react'
import { MyServices } from './MyServices'

export default {
  title: 'Components/MyServices',
  component: MyServices,
} as Meta

export const Default: Story = () => <MyServices />
