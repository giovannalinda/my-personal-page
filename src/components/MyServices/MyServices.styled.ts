import styled from 'styled-components'
import { theme } from '~/styles'

export const Container = styled.section`
  margin-top: 40rem;
  display: flex;
  align-items: center;
  justify-content: center;
  margin-bottom: 40rem;
`

export const Title = styled.h1`
  font-size: ${theme.font.sizes.big};
  text-align: center;

  @media screen and (max-width: ${theme.breakpoints.smallDesktop}) {
    font-size: ${theme.font.sizes.title};
  }

  span {
    color: ${theme.colors.white};
    -webkit-text-fill-color: ${theme.colors.black};
    -webkit-text-stroke-width: 1px;
    -webkit-text-stroke-color: ${theme.colors.white};
  }
`
