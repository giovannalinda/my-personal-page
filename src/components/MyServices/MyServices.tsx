import * as S from './MyServices.styled'

export function MyServices() {
  return (
    <S.Container>
      <S.Title>
        Design <span>&</span> Desenvolvimento
      </S.Title>
    </S.Container>
  )
}
