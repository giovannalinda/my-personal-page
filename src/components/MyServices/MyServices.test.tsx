import { render } from '~/__helpers__/app-tests'
import { MyServices } from './MyServices'

describe('MyServices', () => {
  it('matches snapshot', () => {
    const { container } = render(<MyServices />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
