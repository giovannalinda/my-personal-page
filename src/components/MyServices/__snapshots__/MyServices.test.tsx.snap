// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyServices matches snapshot 1`] = `
<section
  class="sc-bczRLJ gIwltK"
>
  <h1
    class="sc-gsnTZi bqgLsg"
  >
    Design 
    <span>
      &
    </span>
     Desenvolvimento
  </h1>
</section>
`;
